accounting/create_db.py

```diff
@@ -20,10 +20,14 @@
 
 
 def create_db(filepath):
-    # open connection to database
-    logging.info("Creating Flux Accounting DB")
-    conn = sqlite3.connect("file:" + filepath + "?mode:rwc", uri=True)
-    logging.info("Created Flux Accounting DB sucessfully")
+    try:
+        # open connection to database
+        logging.info("Creating Flux Accounting DB")
+        conn = sqlite3.connect("file:" + filepath + "?mode:rwc", uri=True)
+        logging.info("Created Flux Accounting DB successfully")
+    except sqlite3.OperationalError as e:
+        logging.error(e)
+        sys.exit(-1)
```

Comment:
is there a reason for exiting -1 instead of 1?  is it a python thing?

Reply:
Yeah, I also exited -1 here in **accounting_cli.py**:

```python
# try to open database file; will exit with -1 if database file not found
path = args.path if args.path else "FluxAccounting.db"
try:
    conn = sqlite3.connect("file:" + path + "?mode=rw", uri=True)
except sqlite3.OperationalError:
    print("Unable to open database file")
    sys.exit(-1)
```

Reply:
i would say just do `sys.exit(1)` for errors.  I think it's the standard convention unless there's a special python convention I'm unaware of: https://tldp.org/LDP/abs/html/exitcodes.html